wordpress-mobile/gutenberg-mobile
# Disable Autocomplete in shortcode

Fixes: https://github.com/wordpress-mobile/gutenberg-mobile/issues/1808
Gutenberg PR: https://github.com/WordPress/gutenberg/pull/19848

To test:
Tested in Gutenberg-mobile demo app

1. Open the app
2. Add a shortcode block
3. Start editing
4. Expect that the autocomplete suggestions are not displayed

| iOS | Android |
| --- | --- |
| <img width="300" alt="iOS" src="https://user-images.githubusercontent.com/3384451/73005686-5dd93380-3dd7-11ea-84ce-f1288af2bee6.png"> | <img width="300" alt="Android" src="https://user-images.githubusercontent.com/3384451/73005715-6598d800-3dd7-11ea-8cf2-6b295e2de72d.png"> |

PR submission checklist:
